easybuild/easyconfigs/i/imkl/imkl-11.3.3.210-iompi-2016.07.eb

```diff
@@ -0,0 +1,36 @@
+name = 'imkl'
+version = '11.3.3.210'
+
+homepage = 'http://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized, extensively threaded math routines
+for science, engineering, and financial applications that require maximum performance. Core math functions include
+BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'iompi', 'version': '2016.07'}
+
+checksums = ['f72546df27f5ebb0941b5d21fd804e34']
```

Comment:
@jhein32 I don't want to be too picky, but do you mind moving this down below `sources`?

Changes look good otherwise, I'll look into a test report but I don't expect any problems, should be good to go.